src/api/trace/trace_context_propagator.rs

```diff
@@ -171,20 +152,21 @@ impl api::TextMapFormat for TraceContextPropagator {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::api::{Extractor, TextMapFormat};
+    use crate::api::{Extractor, Injector, TextMapFormat};
     use std::collections::HashMap;
+    use std::str::FromStr;
 
     #[rustfmt::skip]
-    fn extract_data() -> Vec<(&'static str, api::SpanContext)> {
+    fn  extract_data() -> Vec<(&'static str, &'static str, api::SpanContext)> {
```

Comment:
this first whitespace was probably not intended.

Reply:
Ooops, serves me right for skipping formatting on this function 😞